.github/scripts/therock_matrix.py

```diff
@@ -0,0 +1,9 @@
+"""
+This dictionary is used to map specific file directory changes to the corresponding build flag and tests
+"""
+monorepo_map = {
+    "projects/rocprim": {
+        "flag": "-DTHEROCK_ENABLE_PRIM=ON -DTHEROCK_ENABLE_ALL=OFF",
```

Comment:
NIT, could be `flags` or `cmake_options`:
```suggestion
        "cmake_options": "-DTHEROCK_ENABLE_PRIM=ON -DTHEROCK_ENABLE_ALL=OFF",
```

Reply:
I like cmake_options a lot more, updating!